cdap-api/src/main/java/co/cask/cdap/api/metrics/RuntimeMetrics.java

```diff
@@ -14,7 +14,7 @@
  * the License.
  */
 
-package co.cask.cdap.test;
+package co.cask.cdap.api.metrics;
```

Comment:
In general I hate the idea of using this ill-designed RuntimeMetrics class as an API for metrics. It raised questions like: how do I access different (or custom) metric in test? How do I wait on it? Etc. We want to revisit the design.

We have multiple jiras for better access to metrics in unit-tests and such - It is fine to revisit this later.